docs/rules/aws_resource_tags.md

```diff
@@ -0,0 +1,47 @@
+# aws_resource_tags
+
+Rule for resources tag presence and value validation from prefixed list.
```

Comment:
```suggestion
Enforce required tag keys and restrict tag values.
```

"Prefixed" connotes a string prefix whereas I'm pretty sure you just mean fixed/exact.